vcbuild.bat

```diff
@@ -14,10 +14,11 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=
 set target=Build
+set target_arch=ia32
+set target_env=vs2015
```

Comment:
can we call this vslegacy or something like that instead? at a glace it looks like I could set it to vs2013 if I wanted that, but that's not the case.